src/Util/Mimetype.php

```diff
@@ -0,0 +1,1084 @@
+<?php
+/**
+ * @file src/Util/Mimetype.php
+ */
+namespace Friendica\Util;
+
+class Mimetype
+{
+	/**
+	 * Return mimetype based on file extension
+	 *
+	 * @param string $filename filename
+	 * @return mixed array or string
+	 */
+	public static function getContentType($filename)
+	{
+		$mime_types = [
```

Comment:
Good, good.

Reply:
Knew you'd like that. After I saw your **huge** PR, figured I'd better roll with it!